src/test/regress/sql/citus_internal_access.sql

```diff
@@ -0,0 +1,7 @@
+--- Create a non-superuser role and check if it can access citus_internal schema functions
+CREATE USER nonsuperuser CREATEROLE;
```

Comment:
This needs to be dropped at the end of the test file, that's why flaky test detection is currently failing in CI. (i.e. it's not flaky, but it's not possible to execute this test multiple times).

Reply:
Thanks, I saw that from the CI logs too. The last commit should solve this.